packages/front-end/pages/features/index.tsx

```diff
@@ -219,12 +239,18 @@ export default function FeaturesPage() {
                 let version = feature.revision?.version || 1;
                 if (isDraft) version++;
 
-                return (
+                return !(!feature.archived || showArchived) ? null : (
                   <tr key={feature.id}>
                     <td>
-                      <Link href={`/features/${feature.id}`}>
-                        <a>{feature.id}</a>
-                      </Link>
+                      {feature.archived ? (
```

Comment:
These two `<Link>` components are almost identical.  I would just render a single component and move the ternary into the `style` prop:

```tsx
<Link href='...'>
  <a style={feature.archived ? {color: "red"} : null}>...</a>
</Link>
```

Reply:
Changed in recent commit